docs/source/examples/elasticsearch_outlet.rst

```diff
@@ -0,0 +1,85 @@
+ElasticSearch Outlet
```

Comment:
Being picky, I know 🙈 but it's `Elasticsearch` actually

Reply:
Yep totally right, I do this ALL the time. Refactoring this everywhere including the example outlet itself.